packetbeat/_meta/kibana/6/dashboard/Packetbeat-dhcpv4.json

```diff
@@ -258,7 +258,8 @@
           "dhcpv4.transaction_id",
           "dhcpv4.op_code",
           "dhcpv4.option.message_type",
-          "client_ip",
+          "source.ip",
+          "destination.ip",
```

Comment:
Nit: shouldn't you use `client.ip` and `server.ip` here, to be better aligned with past naming, and the "data transfer" visualization?

Unless it's semantically important to use those because of the mapping src/dst vs cli/srv flipping around?

Reply:
Using source/destination makes more sense to me because this dataset produces unidirectional flows. The table essentially shows the traffic at a packet level so with `source` and `destination` you can see that A sent this message to B and then know that B responded to A. When I use client/server for a uni-flow I'm not 100% which direction the communication is going since the client/server fields remain the same for all events.

Reply:
Makes sense. Would it make sense to align the "Data transfer" visualization to use source and destination, then?

In any case, I'm all good with this. 👍

Reply:
No it wouldn't because for a uni-flow only the `source` has `bytes`. I had the same thought.